src/site/content/en/blog/bfcache/index.md

```diff
@@ -303,7 +303,15 @@ onAllChangesSaved(() => {
 {% endCompare %}
 
 
-### Avoid window.opener references
+### Do not serve pages with `Cache-control: no-store`
+
+`Cache-control: no-store` is an HTTP header web servers can set on responses to ensure the response is not stored in any cache. When this is set on the page itself (as opposed to any subresource), this means that page will not be included the bfcache. There is [a proposal to change this](https://bugs.chromium.org/p/chromium/issues/detail?id=1228611) but at present any pages sending this are ineligible to use the bfcache.
+
+For sites that simply want to ensure an up-to-date page is used each time, using `Cache-Control: no-cache` or `Cache-Control: max-age=0` prevents the page being used without being revalidated but, unlike `Cache-control: no-store`, still allows the bfcache to be used. For bfcache navigations a page refresh may not be expected or desirable for the user.
+
+For best performance, it is recommended that pages are cacheable, even if only for a short time. This will improve performance in general, but also avoid your site becoming ineligible for the bfcache and therefore not benefiting from those instant page restores. Where this is not possible `Cache-Control: no-cache` or `Cache-Control: max-age=0` should be used unless there are good security reasons for using `Cache-control: no-store`.
```

Comment:
I'm not sure this paragraph is necessary. It mostly just restates the previous two paragraphs.

Instead, it might be good to end with a code example explaining how a developer could check to ensure the user is still logged in a `pageshow` event callback before allowing a page to be restored from bfcache, e.g.:

```js
addEventListener('pageshow', (event) => {
  if (
    event.persisted &&
    // Check to see if the user is still logged in....
    document.cookie.match(/.../)
  ) {
    // Force reload if the user is not logged in...
    location.reload();
  }
});
```

Reply:
Hmmm, apparently I forgotten that [I already added this code](https://web.dev/bfcache/#never-use-the-unload-event:~:text=The%20following%20code%20checks%20for%20the%20presence%20of%20a%20site%2Dspecific%20cookie%20in%20the%20pageshow%20event%20and%20reloads%20if%20the%20cookie%20is%20not%20found%3A).

Reply:
Gave reasons for the two paragraphs above so late me know if you still feel the same.

Agree, it's probably a good idea to link to the https://web.dev/bfcache/#update-stale-or-sensitive-data-after-bfcache-restore section so will add that once we decide if keeping these or not.